source/operations/generic.rst

```diff
@@ -27,6 +27,14 @@ Force NTP sync
    200 OK
    200 OK
 
+you can use osism custom playbook, at ``environments/custom/playbook-force-timesync.yml``, with https://github.com/osism/osism-ansible/blob/master/playbooks/generic-chrony-force-sync.yml
```

Comment:
This is not necessary. I added this playbook to the osism-ansible image.

Reply:
ok, thanks for this information, I delete this PR and branch